DOCS-2336 and DOCS-2369 Synthetics CI/CD Testing Edits (#11447)
DataDog/documentation: content/en/synthetics/api_tests/tcp_tests.md

## Overview

> TCP tests allow you to easily monitor whether or not low-level TCP connections can be established on the ports of given hosts, ensuring the availability of several key services such as `SSH` (22), `SMTP` (25), `DNS` (53), VPN over `HTTPS` (443), and any custom services living on other ports. With built-in response time data, you can keep track of the performance of your network applications and be alerted in case of unexpected slowness. 

Reviewer: "With built-in response time data" what do you mean by that?

Author: I'm not sure exactly! Should we remove it? @HugoPct

Reviewer: Indeed you can keep it.

+ I'm not that fan of "whether or not". A bit negative. 
Could it be replace by "if" ? Or something where we do not offer the "yes or not" schema

TCP tests can run from [managed][1] and [private locations][2] depending on whether you want to launch your TCP connections from outside or inside your network.

## Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][3] can create, edit, and delete Synthetic TCP tests. To get create, edit, and delete access to Synthetic TCP tests, upgrade your user to one of these two [default roles][3].

If you have access to the [custom role feature][4], add your user to a custom role that includes `synthetics_read` and `synthetics_write` permissions for Synthetic Monitoring.

## Configuration

> Now that you have chosen to create a [`TCP` test][5], define your test's request.

Reviewer 2: ```suggestion
After choosing to create a [`TCP` test][5], define your test's request.
```
